### PR TITLE
Quasar 17.8.0 plat 22731 2

### PR DESCRIPTION
**Pull request by danbar0kal**

### CODE REVIEW

File: deployment/permissions/service.baseentry.ini [hunk 1]
```diff
@@ -232,3 +232,12 @@ permissionItem26.param4 =
 permissionItem26.param5 =
 permissionItem26.tags =
 permissionItem26.permissions = CONTENT_MANAGE_BASE, KMC_READ_ONLY, -1>BATCH_BASE, -1>PARTNER_-1_GROUP_*_PERMISSION, BASE_USER_SESSION_PERMISSION, WIDGET_SESSION_PERMISSION, PLAYBACK_BASE_PERMISSION, -6>PARTNER_-6_GROUP_*_PERMISSION
+
+permissionItem27.service = baseentry
+permissionItem27.action = serveplaybackkey
+permissionItem27.partnerId = 0
+permissionItem27.param3 =
```
Comment (MosheMaorKaltura) [inline deployment/permissions/service.baseentry.ini:239]: remove the unset values param3/5 and tags

File: deployment/updates/scripts/add_permissions/2021_08_10_service_baseentry.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (MosheMaorKaltura) [inline deployment/updates/scripts/add_permissions/2021_08_10_service_baseentry.php:1]: rename file to deployment/updates/scripts/add_permissions/2021_08_10_baseentry_add_permission.php

File: release-notes.md [hunk 3]
```diff
@@ -16,12 +16,14 @@
...
 	moduls.enableStaticContentConversion.basePermissionName =
 	moduls.enableStaticContentConversion.group = GROUP_ENABLE_DISABLE_FEATURES
+  
 
 ### Deployment Scripts ### 
 	php deployment/updates/scripts/2021_07_26_deploy_capture_tools_content_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_kaltura_meeting_recordings_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_msft_teams_recordings_conversion_data.php
 	php deployment/updates/scripts/2021_07_26_deploy_zoom_recordings_conversion_data.php
+    php /opt/kaltura/app/deployment/updates/scripts/add_permissions/2021_08_03_service_baseentry.php
```
Comment (MosheMaorKaltura) [inline release-notes.md:26]: You should add new section for this item.
Adding Jira, describing the feature + deployment script.

File: api_v3/services/BaseEntryService.php [hunk 4]
```diff
@@ -1041,5 +1041,30 @@ function getPlaybackContextAction($entryId, KalturaPlaybackContextOptions $conte
 
 		return $result;
 	}
+	
+	/**
+	 * This action serves HLS encrypted key if access control is validated
+	 * @action servePlaybackKey
+	 * @param string $entryId
+	 * @ksOptional
+	 * @return file
```
[the remark this answers is not in the document]
Comment (danbar0kal) [reply api_v3/services/BaseEntryService.php:1050]: per Eran's request (in the chat message you sent me)

File: infra/web/MimeTypes.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php
+class MimeTypes
```
Comment (inbal-ben-david) [inline infra/web/MimeTypes.php:2]: rename to kMimeTypes in the file name and calling function

File: release-notes.md [hunk 6]
```diff
@@ -5,6 +5,14 @@
 * Issue Type: Task
 * Issue ID: FOUN-174
 
+## Add action for serving Playback key ##
```
Comment (inbal-ben-david) [inline release-notes.md:8]: move release notes to 17.10

**Pull request merged**